test-suite/js/src/test/scala/org/scalajs/testsuite/library/UnionTypeTest.scala

```diff
@@ -7,17 +7,11 @@
 \*                                                                      */
 package org.scalajs.testsuite.library
 
-import scala.language.implicitConversions
-
 import scala.collection.mutable
-
+import scala.language.implicitConversions
 import scala.scalajs.js
-import js.|
-
-import org.scalajs.jasminetest.JasmineTest
-import org.scalajs.testsuite.Typechecking._
-
-import js.JSConverters._
+import scala.scalajs.js.JSConverters._
+import scala.scalajs.js.|
```

Comment:
These changes make it impossible to compile the file. Please revert this part as it was before.